src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -42,21 +48,42 @@ public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAl
             _keepAliveDuration = (int)keepAliveInterval.TotalMilliseconds * (Stopwatch.Frequency / 1000);
         }
 
+        /// <summary>
+        /// Gets a <see cref="CancellationToken"/> that notifies when the connection is aborted.
+        /// </summary>
         public virtual CancellationToken ConnectionAborted { get; }
 
+        /// <summary>
+        /// Get the ID for this connection.
+        /// </summary>
         public virtual string ConnectionId => _connectionContext.ConnectionId;
 
+        /// <summary>
+        /// Gets the user for this connection.
+        /// </summary>
         public virtual ClaimsPrincipal User => Features.Get<IConnectionUserFeature>()?.User;
 
+        /// <summary>
+        /// Gets the collection of HTTP features available on this connection.
```

Comment:
~~HTTP~~

They aren't guaranteed to be HTTP features, particularly if/when we support raw TCP or other non-http protocols.